swift-mode.el

```diff
@@ -693,6 +693,115 @@ We try to constraint those lookups by reasonable number of lines.")
                "warning: " (message) line-end))
      :modes swift-mode))
 
+;; BUILDING
+
+(defun chomp (str)
```

Comment:
You should not define unnamespaced functions. Btw, `subr-x.el` has exactly what you need.